src/ecs-mcp-server/awslabs/ecs_mcp_server/modules/troubleshooting.py

```diff
@@ -64,13 +63,128 @@ def register_module(mcp: FastMCP) -> None:
     @mcp.tool(
         name="ecs_troubleshooting_tool",
         annotations=None,
-        description=TROUBLESHOOTING_DOCS,  # Dynamically generated documentation string
     )
     async def mcp_ecs_troubleshooting_tool(
         app_name: Optional[str] = None,
         action: TroubleshootingAction = "get_ecs_troubleshooting_guidance",
         parameters: Optional[Dict[str, Any]] = None,
     ) -> Dict[str, Any]:
+        """
```

Comment:
I need to manually put this long py docs here. Before, we dynamically injected it, but now without the "description" parameter, it is not possible. 

my_tool.__doc__ = MY_DOCS does not work either.

As a temporary hot-fix, we are hard-coding the tool description. A future refactoring can be done to improve this and/or separate this into individual tools. We consolidated them in the first place to reduce the overall number of tools